### DISCUSSION TITLE
About publishing on other websites

### DISCUSSION THREAD

**Discussion by Z1woO** [Aelto/W3_RandomEncounters_Tweaks#70] [General]
I translated mod into simplified Chinese and traditional Chinese. But I seldom use GitHub, and it's troublesome for me to open this website. Can I publish these files elsewhere? I will indicate that this mod comes from you and this website. If it's a problem for you, I won't release it. @Aelto

**Comment by Aelto** [comment]
I see no issue at all as long as the files you provide don't include the RER files! So if alongside your translation you provide a link to the repository it is perfect! Pretty much how people do on nexusmods  where they only share the `.w3strings` file.

Thanks for taking the time to make a translation 👍

**Reply by Z1woO** [reply]
I see what you mean. I only provided the translated documents, and indicated the download address and author of the mod. I hope more people can support you to improve the mod, because you did a great job. I really like the RER.
